scripts/mytop.sh

```diff
@@ -766,6 +796,57 @@ while (1)
         }
     }
 
+    ## w - change columns width for the "User" and "Database" columns
+
+    if ($key eq 'w')
+    {
+        ReadMode($RM_RESET);
+        print RED(), "Width for the 'User' column (the actual value is ".$config{usercol_width}."): ";
+        my $readWidth = ReadLine(0);
+        chomp($readWidth);
+        if ($readWidth ne "" && $readWidth > 4 && $readWidth < 60)
+        {
+            $config{usercol_width} = $readWidth;
+        }
+        elsif ($readWidth ne "")
+        {
+            print RED(), "-- Invalid value ($readWidth), the previous value has been kept, press a key to resume --";
+            ReadKey(0);
+        }
+        print RESET(), RED(), "Width for the 'DB' column (the actual value is ".$config{dbcol_width}."): ", RESET();
+        $readWidth = ReadLine(0);
+        chomp($readWidth);
+        if ($readWidth ne "" && $readWidth > 2 && $readWidth < 60) {
+            $config{dbcol_width} = $readWidth
+        }
+        elsif ($readWidth ne "")
+        {
+            print RED(), "-- Invalid value ($readWidth), the previous value has been kept, press a key to resume --", RESET();
+            ReadKey(0);
+        }
+        undef $readWidth;
+        ReadMode($RM_NOBLKRD);
+        next;
+    }
+
+    ## a - progress column toggle (the column is only displayed if progress informations are available from the processlist)
+
+    if ($key eq 'a')
+    {
+        if ($config{hide_progress})
+        {
+            $config{hide_progress} = 0;
+            print RED(), "-- progress display ON --", RESET();
+            sleep 1;
+        }
+        else
+        {
+            $config{hide_progress} = 1;
+            print RED(), "-- progress display OFF --", RESET();
+            sleep 1;
+        }
+    }
+
```

Comment:
Can we also change L:866 to use `system("cls")` ?

Reply:
It should be tested as it might not work as `clear` on Linux.

Reply:
Yes we should test this also.